samples/rx65ncloudkit/.gitignore

```diff
@@ -22,8 +22,7 @@
 /iotc-azrtos-sdk/cJSON/
```

Comment:
You probably wanted to remove this cJSON from sdk, as it will get pulled in by deps